examples/reconfigurationBeam/circularReconfiguration.m

```diff
@@ -0,0 +1,224 @@
+%% Add in assembler at 338
+
+%  global FDrag
+% FDrag(timeVar) = sum(Faero(1:6:end)) ;
+% # Reconfiguration problem validation (Drag reduction of flexible plates by reconfiguration, Gosselin, etAl 2010)
+%----------------------------
+close all, clear all ;
+% add path
+addpath( genpath( [ pwd '/../../src'] ) ); tic;
```

Comment:
```suggestion
addpath( genpath( [ pwd '/../..'] ) ); tic;
```

Reply:
doing this change. in this example you load the entire onsas path, including the examples--- and I think that the .mat file will be in the path then...